neuralprophet/forecaster.py

```diff
@@ -1664,10 +1672,9 @@ def plot(
         plotting_backend = (
```

Comment:
if plotting backend is not given by user here (set to default/auto):
- check if it was previously set (if so, use that)
- if not, call function auto_set_plotting_backend(), get one of 3 backends, depending on env. 

if plotting backend is given by user (in a plot function), use, but only for this call (self.plotting_backend)

if user calls set_plotting_backend():
- if default/auto, call function auto_set_plotting_backend(), get one of 3 backends, depending on env. Then write this value to self.plotting_backend
-else, if one of the 3 backends,  writes value to self.plotting_backend